### PR TITLE
V0.18.0

### PR DESCRIPTION
**Pull request by swanderz**
Upgrade adapter to be compatible with `dbt` version `0.18.0`

### CODE REVIEW

File: dbt/include/sqlserver/macros/materializations/snapshot/snapshot.sql [hunk 1]
```diff
@@ -1,206 +1,4 @@
...
-    {% call statement() %}
-      alter table {{ relation }} add "{{ column.name }}" {{ column.data_type }};
-    {% endcall %}
-  {% endfor %}
-{% endmacro %}
\ No newline at end of file
+{% macro sqlserver__post_snapshot(staging_relation) %}
+  -- Clean up the snapshot temp table
+  {% do drop_relation(staging_relation) %}
+{% endmacro %}
```
Comment (swanderz) [inline dbt/include/sqlserver/macros/materializations/snapshot/snapshot.sql:4]: my understanding is that in removing all these custom snapshots macros, that the default macros are now compatible with `sqlserver`?

Comment (mikaelene) [reply dbt/include/sqlserver/macros/materializations/snapshot/snapshot.sql:4]: Yes! Most of them are :-).

Comment (swanderz) [reply dbt/include/sqlserver/macros/materializations/snapshot/snapshot.sql:4]: ok awesome. this definitely makes things simpler for @nandanhegde15 and I with `dbt-synapse`. Right now `MERGE` is in only private preview for Synapse, so we have to hack our own version of the `snapshot_merge_sql()` macro...

**Comment by mikaelene** [comment]
v0.18.0 seems to work fine so I am merging this to be the new master!

**Pull request merged**